### PR TITLE
Added the Unity Skeleton Package and updated the main ReadMe with information pertaining to it.

### PR DESCRIPTION
**Pull request by mcengija**
This is the unity package to make it easy for a user to add skeletons to their existing unity project.

### CODE REVIEW

File: unityPackage/Runtime/Scripts/Rig_Bridge/test/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+project( rig2csTest CSharp)
```
Comment (johnhe4) [inline unityPackage/Runtime/Scripts/Rig_Bridge/test/CMakeLists.txt]: I recognize this as the rig2csTest already in the repository. Is this something different?

File: unityPackage/Runtime/Prefabs/SkeletonLoader.prefab [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+    header:
+      endFrame: 0
+      fps: 0
+      startFrame: 0
+  rigsData:
+    rigs: []
+  originalFrame: 0
+  CurrentFrame: 0
+  FrameCount: 0
+  url: NFL/seg_91888.json
```
Comment (johnhe4) [inline unityPackage/Runtime/Prefabs/SkeletonLoader.prefab]: Please rename this URL (folder, specifically), or remove the value if it doesn't need to be set

File: unityPackage/Runtime/Scripts/Rig_Bridge/test/CMakeLists.txt (hunk 1 above)
Comment (mcengija) [reply unityPackage/Runtime/Scripts/Rig_Bridge/test/CMakeLists.txt]: I removed the whole folder and updated the ReadMe to include steps to add your rig2cs folder to the users Unity Project.

File: unityPackage/Runtime/Prefabs/SkeletonLoader.prefab (hunk 2 above)
Comment (mcengija) [reply unityPackage/Runtime/Prefabs/SkeletonLoader.prefab]: made the URL field empty.

**Review by johnhe4 — APPROVED** [review]

**Pull request merged**